calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -7413,8 +7413,14 @@ protected void handleAccessOpaqueUrl(HttpServletRequest req, HttpServletResponse
 			}
 		}
 	}
-	
-	protected List<String> getCalendarReferences(String siteId) {
+
+	/**
+	 * Made this method public as the same code was repeated in CalendarBean.java and
```

Comment:
This shoudn't be in the JavaDoc, but in the commit comment, the JavaDoc should describe what the method does.